examples/apps/hugging_face_integration_app/med_image_generation/app.py

```diff
@@ -0,0 +1,39 @@
+import logging 
+from pathlib import Path
+import torch 
+from diffusers import StableDiffusionPipeline
+from monai.deploy.core import AppContext, Application
+from PIL import Image
+import numpy as np
+import argparse
+
+
+
+class App(Application):
+	name = "Diffusion Image App"
+	description = "Simple application showing diffusion to generate Images"
+	def compose(self):
+		model_id = "Nihirc/Prompt2MedImage"
```

Comment:
Is this a local file? Can it use the common option for passing model path?

Reply:
No it is not a local file It is the model id. 
https://huggingface.co/Nihirc/Prompt2MedImage